testing/internal/e2e/tests/base_with_vault/auth_method_oidc_vault_test.go

```diff
@@ -312,4 +319,260 @@ func TestAuthMethodOidcVault(t *testing.T) {
 		),
 	)
 	require.NoError(t, output.Err, string(output.Stderr))
+
+	// Set new auth method as primary auth method for the new org
+	output = e2e.RunCommand(ctx, "boundary",
+		e2e.WithArgs(
+			"scopes", "update",
+			"-id", orgId,
+			"-primary-auth-method-id", authMethodId,
+			"-format", "json",
+		),
+	)
+	require.NoError(t, output.Err, string(output.Stderr))
+	var updateScopeResult scopes.ScopeUpdateResult
+	err = json.Unmarshal(output.Stdout, &updateScopeResult)
+	require.NoError(t, err)
+	require.Equal(t, authMethodId, updateScopeResult.Item.PrimaryAuthMethodId)
+
+	// Create managed group
+	output = e2e.RunCommand(ctx, "boundary",
+		e2e.WithArgs(
+			"managed-groups", "create", "oidc",
+			"-auth-method-id", authMethodId,
+			"-name", groupName,
+			"-filter", fmt.Sprintf(`"%s" in "/userinfo/groups"`, groupName),
+			"-format", "json",
+		),
+	)
+	require.NoError(t, output.Err, string(output.Stderr))
+	var managedGroupCreateResult managedgroups.ManagedGroupCreateResult
+	err = json.Unmarshal(output.Stdout, &managedGroupCreateResult)
+	require.NoError(t, err)
+	managedGroupId := managedGroupCreateResult.Item.Id
+	t.Logf("Created Managed Group: %s", managedGroupId)
+
+	// Start OIDC authentication process to Boundary
+	t.Log("Authenticating using OIDC...")
+	res, err := http.Post(
+		fmt.Sprintf("%s/v1/auth-methods/%s:authenticate", boundary.GetAddr(t), authMethodId),
+		"application/json",
+		strings.NewReader(
+			fmt.Sprintf(`{"command": "start"}`),
+		),
+	)
+	require.NoError(t, err)
+	t.Cleanup(func() {
+		res.Body.Close()
+	})
+	require.Equal(t, http.StatusOK, res.StatusCode)
+	var authResult authmethods.AuthenticateResult
+	err = json.NewDecoder(res.Body).Decode(&authResult)
+	require.NoError(t, err)
+	oidcTokenId := authResult.Attributes["token_id"].(string)
+	authUrl := authResult.Attributes["auth_url"].(string)
+	u, err := url.Parse(authUrl)
+	require.NoError(t, err)
+	m, _ := url.ParseQuery(u.RawQuery)
+	nonce := m["nonce"][0]
+	state := m["state"][0]
+
+	// Vault: Authenticate to get a client token
+	res, err = http.Post(
+		fmt.Sprintf("%s/v1/auth/userpass/login/%s", c.VaultAddr, userName),
+		"application/json",
+		strings.NewReader(
+			fmt.Sprintf(`{"password": "%s"}`, userPassword),
```

Comment:
Note that this will break if `userPassword` contains `"` (not that it's likely to but). 
```suggestion
			fmt.Sprintf(`{"password": %q}`, userPassword),
```
Should be slightly safer 😅